### DISCUSSION TITLE
标签命名修改意见征集

### DISCUSSION THREAD

**Discussion by xaoxuu** [xaoxuu/hexo-theme-stellar#198] [Ideas]
有些标签命名不是很合适，希望征集大家意见找个合适的名字：

## border
这个标签最初命名 `noteblock` 因 `6.x` 和官方冲突被迫改名，现在改了几个都不太合适。
标签特点：有边框和背景的容器标签。

## about
这个标签最初就是给关于页面顶部做的一个样式，后越来越通用化了。
标签特点：无边框有背景和大圆角的容器标签，内部的padding较大

## split
这个标签一直没想到合适的命名
标签特点：桌面端双列布局的容器标签（移动端因宽度有限显示为单列布局）
> 或许可以叫 grid ？

**Comment by shaoyaoqian** [comment]
改了以后之前的文章是不是要全部重写？

**Reply by xaoxuu** [reply]
不需要，全局替换一下就好了，假如 `split` -> `grid` 的话，替换：
- `{% split` -> `{% grid`
- `{% endsplit` -> `{% endgrid`

**Reply by panoshu** [reply]
在shell里面用sed命令批量替换，很方便的。假如 split -> grid 的话，替换
```bash
sed -i "s/{% split/{% grid/g"  `grep '{% split' -rl ./source/*`
sed -i "s/{% split/{% endgrid/g"  `grep '{% endsplit' -rl ./source/*`
```
